### ISSUE TITLE
Adding decoder debug to wazuh-logtest verbose mode

### ISSUE THREAD

**Issue by JcabreraC** [wazuh/wazuh#9206]
Hello team,

This issue is to add a new feature to X. The idea is to be able to follow the correlation intent trace of a log with the decoders as it is currently done with the rules.

**Comment by branchnetconsulting** [comment]
Yes, I've wished for this exact thing for years.  Some kind of tracing of the decoder stage.  It may be a little tricky to work out with the way that child decoders have no description field nor are their names always unique from their parent, but something would be better than nothing.
